dataservice/api/errors.py

```diff
@@ -1,6 +1,45 @@
 from dataservice.api.common.schemas import ErrorSchema
 
+FOREIGN_KEY_ERROR_PREFIX = 'foreign key constraint failed'
```

Comment:
I know this is duplicate code with demographic-entity branch. Just needed to include this since that branch has not been merged in and it is required to implement the diagnosis resource module.

Reply:
should be taken care of during rebase